cmd/ui-errors.go

```diff
@@ -0,0 +1,169 @@
+		"Please provide an even number of endpoints greater or equal to 4",
+		"",
+	)
+
+	uiErrStorageClassValue = newUIErrFn(
+		"Invalid storage class value",
+		"Please check the passed value",
+		`MINIO_STORAGE_CLASS_STANDARD: Format "EC:<Default_Parity_Standard_Class>" (e.g. "EC:3"). This sets the number of parity disks for Minio server in Standard mode. Object are stored in Standard mode, if storage class is not defined in Put request.
+MINIO_STORAGE_CLASS_RRS: Format "EC:<Default_Parity_Reduced_Redundancy_Class>" (e.g. "EC:3"). This sets the number of parity disks for Minio server in reduced redundancy mode. Objects are stored in Reduced Redundancy mode, if Put request specifies RRS storage class.`,
+	)
+
+	uiErrUnexpectedBackendVersion = newUIErrFn(
+		"Backend version seems to be too recent",
+		"Please update to the latest Minio version",
+		"",
+	)
+
+	uiErrInvalidAddressFlag = newUIErrFn(
+		"--address parameter is not suitable for use",
+		"Please check --address parameter",
+		`--address binds a specific ADDRESS:PORT, ADDRESS can be an  IP or hostname (default:':9000')
+    Examples: --address ':443'
+              --address '172.16.34.31:9000'`,
+	)
+
+	uiErrInvalidFSEndpoint = newUIErrFn(
+		"The given endpoint is not suitable to activate standalone mode",
+		"Please check the given FS endpoint",
+		`Standalone mode requires only one local disk path to be activated.
+Example 1:
+   $ minio server /var/data/minio/`,
+	)
+
+	uiErrUnableToWriteInBackend = newUIErrFn(
+		"Unable to write on the specified endpoint",
+		"Please ensure that Minio binary has enough permissions to write in the backend",
+		"",
+	)
+
+	uiErrUnableToReadFromBackend = newUIErrFn(
+		"Unable to read data from the backend",
+		"Please ensure that Minio binary has enough permission to read data from the backend",
+		"",
+	)
+
+	uiErrPortAlreadyInUse = newUIErrFn(
+		"Port is already in use",
+		"Please ensure no other program is reserving the same address/port",
+		"",
+	)
+
+	uiErrNoPrivilegesToAccessDirFiles = newUIErrFn(
+		"Missing privileges to access to the specified path",
+		"Please ensure the specified path can be accessed with your privileges",
+		"",
+	)
+
+	uiErrSSLUnexpectedError = newUIErrFn(
+		"Something wrong with your TLS certificate",
+		"Please check the content of your certificate data",
+		`Only PEM (x.509) format is accepted as valid public & private certificates.`,
+	)
+
+	uiErrSSLUnexpectedData = newUIErrFn(
+		"Something wrong with your TLS certificate",
+		"Please check your provided certificate",
+		"",
+	)
+
+	uiErrSSLNoPassword = newUIErrFn(
+		"The private key is protected by a password but no password is specified",
+		"Please set the password to this variable environment `"+TLSPrivateKeyPassword+"` so Minio can decrypt the private",
+		"",
+	)
+
+	uiErrNoCertsAndHTTPSEndpoints = newUIErrFn(
+		"HTTPS is specified in endpoint urls but no certificate is found in the local machine",
```

Comment:
`HTTPS is specified in endpoint URLs but no TLS certificate is found on the local machine`